README.md

```diff
@@ -72,15 +76,25 @@ COMMUTER_PATH_DELIMITER (optional, defaults to "/")
 COMMUTER_PORT (optional, defaults to 4000)
 ```
 
-Project uses [prettier](https://github.com/jlongster/prettier) for code formatting (`npm run format:code` and [package.json] (https://github.com/nteract/commuter/blob/master/package.json) has more options).
+Project uses [prettier](https://github.com/jlongster/prettier) for code formatting (`npm run format:code` and [package.json](https://github.com/nteract/commuter/blob/master/package.json) has more options).
 
 ## Tests
 
 1. `npm test`
 
 ## Deployment
 
-coming soon...
+There are few different ways to get commuter deployed on your severs:
+
+Firstly, to change default port `4000` set env variables `COMMUTER_PORT`
+
+1. If you want to deploy `master` branch then do `git clone git@github.com:nteract/commuter.git` and run `npm run start:production` on your server
```

Comment:
I'd probably make this a fenced block like:

````markdown

If you want to deploy from `master`, run:

```
git clone git@github.com:nteract/commuter.git
cd commuter
npm i
npm run start:production
```

````